packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/confirmationPopup.tests.js

```diff
@@ -0,0 +1,76 @@
+import $ from 'jquery';
+
+import ConfirmationPopup from '__internal/ui/chat/confirmationpopup';
+
+const DX_BUTTON_CLASSNAME = 'dx-button';
+
+const moduleConfig = {
+    beforeEach: function() {
+        this.$element = $('#component');
+    },
+};
+
+QUnit.module('ConfirmationPopup', moduleConfig, () => {
+    QUnit.test('Should execute onApplyButtonClick callback when apply button is clicked', function(assert) {
+        assert.expect(1);
+
+        const done = assert.async();
+        const confirmationPopup = new ConfirmationPopup(
```

Comment:
Lets extract this part to 'reinit(), init()' methods like in other test files done